lib/modules/apostrophe-logger/index.js

```diff
@@ -0,0 +1,19 @@
+const config = require('./config/index');
+
+module.exports = {
+  alias: 'logger',
+  construct: (self, options) => {
+    const clientLoggingProvider = process.env.LOGGING_PROVIDER_CLIENT;
+    const providerConfig = config[clientLoggingProvider];
+
+    self.addHelpers({
+      renderClientProvider: () => {
+        if(!providerConfig) {
```

Comment:
Kunnen we hier een check inbouwen om te kijken of wel alle benodigde instellingen aanwezig zijn in de providerConfig?

Reply:
Goeie, ik heb de code een beetje gerefactored. 
**Let op** ik heb ook de env variables hernoemd :-)